Describe two new `rel` link types. (#29801)
mdn/content: files/en-us/web/html/attributes/rel/index.md

| [`stylesheet`](#stylesheet)                                          | Imports a style sheet.                                                                                                                                                                                                                                                                  | External Resource       | Not allowed                                      | Not allowed             |
| [`tag`](#tag)                                                        | Gives a tag (identified by the given address) that applies to the current document.                                                                                                                                                                                                     | Not allowed             | Link                                             | Not allowed             |

> | [`terms-of-service`](#terms-of-service)                              | Link to the agreement, or terms of service, between the document's provider and users who wish to use the document.                                                                                                                                 | Link                    | Link                                             | Not allowed             |

Author: It's not clear to me what the linter doesn't like in this line. It appears to be suggesting a no-op change?

> | [`terms-of-service`](#terms-of-service)                              | Gives a link to a information about the information about the agreements between a document's provider and users who wish to use the document provided.                                                                                                                                 | Link                    | Link                                             | Not allowed             |

Reviewer: ```suggestion
| [`terms-of-service`](#terms-of-service)                              | Link to the agreement, or terms of service, between the document's provider and users who wish to use the document.                                                                                                                                 | Link                    | Link                                             | Not allowed             |
```
alternatively, "Indicates that the linked resource is the terms of service for the current resource".

The description is linked below. This should be a quick description. See `help` as the example that should be used. 

This table has gotten too verbose. But that is an edit for a different PR.
